src/Nethermind/Nethermind.BeaconNode/MockedStart/QuickStart.cs

```diff
@@ -127,11 +128,13 @@ public void QuickStartGenesis()
 
                 // TODO: Add some tests around quick start, then improve
 
-                var index = depositDataList.Count;
+                int index = depositDataList.Count;
                 depositDataList.Add(depositData);
-                var root = depositDataList.HashTreeRoot((ulong)1 << _chainConstants.DepositContractTreeDepth);
+                Hash32 root = depositDataList.HashTreeRoot((ulong)1 << _chainConstants.DepositContractTreeDepth);
                 var allLeaves = depositDataList.Select(x => x.HashTreeRoot());
                 var tree = CalculateMerkleTreeFromLeaves(allLeaves);
```

Comment:
Yeah, this code is actually all pretty crap, as I copied it from their tests and it recalculates the (mostly the same, one new item) Merkle tree each time. If this was real code it would be a good place to optimise.

However, this is only called once at startup, and only when running in a mocked mode for interop testing, so performance isn't really an issue. 

As an alternative, if loading lots of validators, there is also a file with pre-calculated values for the first 10,000, so we would just implement loading that (in the future).

Reply:
ok, thanks, for now I was looking at the code without understanding the context